src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -0,0 +1,221 @@
+package com.denizenscript.depenizen.bukkit.commands.quests;
+
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+
+import me.blackvein.quests.Quest;
+import me.blackvein.quests.Quester;
+import me.blackvein.quests.Quests;
+import me.blackvein.quests.exceptions.InvalidStageException;
+import me.blackvein.quests.util.Lang;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+
+import org.bukkit.entity.Player;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.depenizen.bukkit.bridges.QuestsBridge;
+
+/**
+ * Command integration for Quests
+ * @author PikaMug
+ * @since 2.0.0-b595
+ */

```

Comment:
I do believe this is something you randomly added for the sake of testing commits to fix the GitHub mislink. Either way, doesn't belong and should be removed.

Reply:
Correct, although if it were a larger commit I'd insist on the @author tag. Will remove.